addons/udes_priorities/models/stock_picking.py

```diff
@@ -37,31 +40,41 @@ def _priority_and_priority_group_domain(self):
 
         return domain
 
-    def _priority_domain(self):
+    def _priority_domain(self, picking_type_ids=None):
         # _priority_domain and _priority_group_domain may diverge in certain circumstances
         # To allow for this they are proxies to the default function
-        return self._priority_and_priority_group_domain()
+        return self._priority_and_priority_group_domain(picking_type_ids=picking_type_ids)
 
-    def _priority_group_domain(self):
+    def _priority_group_domain(self, picking_type_ids=None):
         # _priority_domain and _priority_group_domain may diverge in certain circumstances
         # To allow for this they are proxies to the default function
-        return self._priority_and_priority_group_domain()
+        return self._priority_and_priority_group_domain(picking_type_ids=picking_type_ids)
 
     @api.model
     def get_priorities_for_selection(self):
         Priorities = self.env["udes_priorities.priority"]
-
         active_id = self.env.context.get("active_id", None)
         pick = self
         if active_id:
             active_pick = self.browse(active_id).exists()
             if active_pick:
                 pick = active_pick
 
-        return Priorities.search(pick._priority_domain()).get_selection_values()
+        priorities = Priorities.search(pick._priority_domain())
+
+        # hard coded default value means there is always a priority to set
+        normal = self.env.ref("udes_priorities.normal")
+        priorities |= normal
```

Comment:
Same as above.

Reply:
as we can default to normal we always want to provide that back as a choice, we could run into some nasties where move has defaulted to normal (say we make one without pick type) and the picking we try to give it to doesn't have normal as an acceptable value. This is mostly relevant when you try to set values to nothing, there has to be a supreme default, along the lines of why vanilla odoo always puts 1